tests/job-submission/18-check-chunking.t

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2018 NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+
+# Test that job submission kill on timeout results in a failed job submission.
+
+. "$(dirname "$0")/test_header"
+
+set_test_number 3
+
+init_suite "${TEST_NAME_BASE}" <<'__SUITERC__'
+[cylc]
+    [[parameters]]
+        p = 1..202
+[scheduling]
+    [[dependencies]]
+        graph = t1<p>
+[runtime]
+    [[t1<p>]]
+        script = """
+wait
+sleep $((RANDOM % 5))
+"""
+__SUITERC__
+
+
+TEST_NAME=${TEST_NAME_BASE}-run
+run_ok $TEST_NAME cylc run --debug --no-detach $SUITE_NAME
+LOG_FILE=$(cylc cat-log $SUITE_NAME -m p)
+echo $LOG_FILE >&2
+
+TEST_NAME=${TEST_NAME_BASE}-itasks-msg
+grep_ok "Submitting jobs batch 1/3 (68 task(s))" $LOG_FILE
+grep_ok "Submitting jobs batch 3/3 (66 task(s))" $LOG_FILE
```

Comment:
Do you want to print 2/2 as well?

Reply:
I did not think that it was needed.
But I can do if you feel I should.